### PR TITLE
Tony: Added project section with links

### PR DESCRIPTION
**Pull request by wangtony4005**
Created project section and added link to project. Added links for experience section as well. Changed some spacing and css. Fixed flashlight not going down when scrolling.

L=ENG-111

### CODE REVIEW

File: js/src/personal/tony/Tony.module.css [hunk 1]
```diff
@@ -108,18 +114,38 @@
...
-  margin-right: 10px;
-  height: 200px;
   width: 200px;
+  height: 200px;
+  border-radius: 10px;
 }
 
-.text1{
-  margin-bottom: 15px;
+.Link {
```
Comment (arklian) [inline js/src/personal/tony/Tony.module.css:145]: be consistent in your naming scheme.

File: js/src/personal/tony/Tony.page.tsx [hunk 2]
```diff
@@ -110,48 +114,78 @@ export function TonyPage() {
...
-                }
-              </Text>
-              <Text className={styles.text1}>
-                {
-                  'Provided tutoring for 400+ students in topics related to OOP in C++ and Python. Created and graded projects/assignments in C++. Conducted weekly code reviews.'
-                }
-              </Text>
+              <a
+                href="https://maryash.github.io/135/index.html"
+                rel="noopener noreferrer"
```
Comment (arklian) [inline js/src/personal/tony/Tony.page.tsx:119]: What do these two do btw?

File: js/src/personal/tony/Tony.module.css [hunk 3]
```diff
@@ -1,4 +1,4 @@
-.title {
+.name {
```
Comment (arklian) [inline js/src/personal/tony/Tony.module.css:1]: Add a screenshot

File: js/src/personal/tony/Tony.module.css [hunk 4]
```diff
@@ -10,7 +10,7 @@
   }
 }
 
-.text {
+.textSWE {
   color: #DADDD8;
   -webkit-font-smoothing: antialiased;
```
Comment (arklian) [inline js/src/personal/tony/Tony.module.css:15]: Isn't this like for safari? How'd you land on this?

**Comment by wangtony4005** [comment]
Screenshot 2024-07-01 at 3 49 24 PM

File: js/src/personal/tony/Tony.module.css [hunk 5]
```diff
@@ -57,69 +61,90 @@
...
   margin-top: 16px;
-  margin-right: 4px ;
-  background-color: #ECEBE4;
+  margin-right: 4px;
+  background-color: #ecebe4;
 }
 
-.menu{
+.menu {
   margin-left: -14px;
```
Comment (arklian) [inline js/src/personal/tony/Tony.module.css:73]: Why negative margin

**Review by arklian — DISMISSED** [review]
Fix the negative margin when you can

**Review by HaoxinLuo — APPROVED** [review]

**Pull request merged**